src/styles/blocks/card.scss

```
@@ -0,0 +1,104 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  font-family: "Roboto", sans-serif;
+  color: $main-color;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:not(:nth-last-child(-n + 4)) {
+    margin-bottom: 46px;
+  }
+
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__suptitle {
+    margin-bottom: 16px;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+
+    color: $paragraph-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 16px;
+  }
+
+  &__price-title {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $paragraph-color;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #fff;
+
+    border-radius: 5px;
+    background-color: $secondary-color;
+    border: 1px solid $secondary-color;
+
+    transition: 0.3s all ease;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.3s property1 ease, 0.3s property2 ease;
```

---

done